### ISSUE TITLE
IPT and DwC locations for ParasiteTracker data

### ISSUE THREAD

**Issue by seltmann** [ParasiteTracker/other-projects-modeling-biotic-interactions#6]
@jhpoelen Do you have a list of the IPT and DwC locations for PT data going to GloBI? @evindunn and I are working on modifications in SCAN to upload all of the occurrence data to SCAN for the project, creating a "unified" dataset.

They generally be collection snapshots of all of the PT project data being produced.

**Comment by jhpoelen** [comment]
Happy to provide a list of URLs indexed by GloBI. What do you mean by list of locations of Parasite Tracker (PT) data? The sources of the "green" collections that are listed on https://globalbioticinteractions.org/parasitetracker ?

**Comment by seltmann** [comment]
Yes, the green ones listed at https://globalbioticinteractions.org/parasitetracker

**Comment by jhpoelen** [comment]
Ok, thanks for clarifying. One more question - some of the TPT DwC-A are considered, but do not show up as green because no biotic associations were extracted. I also expect others to be added in the near future from new data sources (e.g., individual IPTs or other RSS feeds) . How are you planning to update the "unified" TPT dataset from "green" DwC-A list?

**Comment by seltmann** [comment]
The arthropod data would become snap shot collections in SCAN even if there is zero interaction data. Some may come from SCAN. These would not need to be included. The data would be updated periodically.
